### PR TITLE
Update to go 1.9.2

### PR DESCRIPTION
**Pull request by jwilder**
###### Required for all non-trivial PRs
- [ ] Rebased/mergable
- [ ] Tests pass
- [ ] CHANGELOG.md updated
- [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Updates to go1.9.2

### CODE REVIEW

**Comment by e-dard** [comment]
@jwilder my only suggestion would be we switch out the 1.9 test runner for a 1.10 one. The 1.10 branch is open. I assume Circle doesn't fail the build if the testing-future-go-version runner fails does it?

**Comment by mark-rushakoff** [comment]
There's a couple other mentions of Go versions that need to be updated. I'm locating those now and I'll drop them in here.

**Comment by jwilder** [comment]
@e-dard It does fail the build.

**Comment by mark-rushakoff** [comment]
@jwilder there's some go 1.8.4 occurrences in CONTRIBUTING.md that need updated, then I think this PR is good to go.

**Comment by jwilder** [comment]
@mark-rushakoff Fixed

File: Dockerfile_build_ubuntu64_go19 [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN gem install fpm
 
 # Install go
 ENV GOPATH /root/go
-ENV GO_VERSION 1.9.1
+ENV GO_VERSION 1.9.2
```
Comment (mark-rushakoff) [inline Dockerfile_build_ubuntu64_go19]: The _go19 file isn't really necessary - it should be _go110 but I'm not sure whether that's available yet. I'm fine leaving the file around so we don't have to undo and redo the work to have circle test the next Go version.

Comment (jwilder) [reply Dockerfile_build_ubuntu64_go19]: Yeah.  I left it as is since I couldn't find a go1.10.

**Comment by e-dard** [comment]
@jwilder dang. I guess we should probably hold the 1.10 runner off until the branch is a bit more stable.

**Pull request merged**